gitpod-io/website
# :rocket: Screencast Pages Revamp

## Description

|Light Mode (from Mockup)|Dark Mode (Creativity of mine 🧪)|
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/55068936/223094844-a160a061-09de-4366-81ed-55465c702903.png)|![image](https://user-images.githubusercontent.com/55068936/223094928-f8caf961-5a50-407e-b0b0-55ec56790b8d.png)|

## Related Issue(s)



Fixes #3453

## How to test



- Open [`/screencasts`](https://deploy-preview-3491--gitpod-kumquat.netlify.app/screencasts) & see :eyes:

## Release Notes



```release-note
NONE
```


<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3491"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
